src/MissionEditor/SimpleItemEditor.qml

```diff
@@ -119,6 +119,13 @@ Rectangle {
                 onClicked:  editorRoot.moveHomeToMapCenter()
                 anchors.horizontalCenter: parent.horizontalCenter
             }
+
+            QGCButton {
```

Comment:
Oh, I thought this was centering around current item. Sorry not looking close enough.  If you want to center around home just use this:
![screen shot 2016-04-13 at 8 44 23 am](https://cloud.githubusercontent.com/assets/5876851/14499635/f97efd84-0153-11e6-9d1a-e949a3251dd5.png)